docs/en/e-service-pdnd.rst

```diff
@@ -0,0 +1,1187 @@
+
+    The Interoperability API includes an event notification endpoint that alerts subscribed Participants about changes within the PDND Infrastructure. Among these notifications, the ``/events/keys`` endpoint provides updates on modifications to cryptographic material, such as additions or deletions of keys. By leveraging this mechanism, Participants can implement a periodic polling strategy to retrieve all changed keys and update their local cache. This eliminates the need to request each key individually during the workflow.
+
+
+.well-known Endpoint
+-------------------------------------
+
+The .well-known Endpoint is part of the PDND Infrastructure and used to retrieve the public keys used by the PDND Authorization Server to sign the Vouchers.
+
+Keys Request
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The Keys Request is a ``GET`` HTTP request sent to the .well-known Endpoint. This endpoint allows Participants to retrieve the public keys necessary to verify digital signatures on Vouchers issued by the PDND Authorization Server.
+
+Keys Response
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The .well-known Endpoint responds with a ``200`` OK status code and a ``JWK Set`` [:rfc:`7517`] containing the public keys employed by the PDND Authorization Server to sign Vouchers.
+
+Interoperability API Endpoint
+-------------------------------------
+
+The Interoperability API Endpoint is part of the PDND Infrastructure and used to retrieve the public keys of other parties enrolled in the PDND.
+
+Key Request
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The Key Request is a ``GET`` HTTP request sent to the ``/keys/<kid>`` API. This request is used to retrieve a specific key identified by its unique ``kid``.
+
+The Key Request MUST include the following HTTP header parameters:
+
+.. list-table::
+    :widths: 20 60 20
+    :header-rows: 1
+
+    * - **Parameter**
+      - **Description**
+      - **Reference**
+    * - **Authorization**
+      - Voucher released by the PDND Authorization Server.
+      - [:rfc:`9449`]
+
+Key Response
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The Interoperability API Endpoint responds with a ``200`` OK status code and a ``JWK`` [:rfc:`7517`] representing the public key with the provided ``kid``.
+
+
+e-Service Usage
+===================
+
+Prerequisites
+----------------------
+
+The **Consumer** MUST comply with the following prerequisites:
+
+    - Has obtained a valid Voucher to interact with the intended e-Service, related to a specific `Client e-service`.
+
+The **Provider** MUST comply with the following prerequisites:
+
+    - Has created a new key ring associated with the specific e-Service.
+    - Has registered a key pair associated with the key ring.
+
+.. note:: 
+
+    The key ring on the Provider's side is the reciprocal entity to the Client on the Consumer's side. This component serves as a store for cryptographic material, thus enabling Consumers to verify the integrity of responses transmitted by Providers.
```

Comment:
```suggestion
    The Provider's key ring is the counterpart to the Client on the Consumer's side. It stores cryptographic material, allowing Consumers to verify the integrity of responses from Providers.
```

I am not sure to have fully understood the sentence above, even if I try to simplify it